evap/staff/tests/test_views.py

```diff
@@ -527,6 +529,23 @@ def test_preparation_reminder(self):
         self.assertContains(response, 'user_to_find')
         self.assertContains(response, 'name_to_find')
 
+    @patch("evap.staff.views.EmailTemplate.send_to_user")
+    def test_remind_all(self, send_to_user_mock):
+        user = mommy.make(UserProfile)
+        evaluation = mommy.make(Evaluation, course=mommy.make(Course, semester=self.semester, responsibles=[user]), state='prepared')
+
+        response = self.app.post(self.url, user='manager')
+        self.assertEqual(response.status_int, 200)
```

Comment:
So far in our code, we always used `status_code`. Is there a difference between the two? If not, I'd like to keep it unified and use `status_code` here as well.